Pod/Classes/TSSwiftMarkdownParser.swift

```diff
@@ -98,6 +100,19 @@ public class TSSwiftMarkdownParser: TSBaseParser {
         strongAndEmphasisAttributes = [NSFontAttributeName: strongAndEmphasisFont]
         
         if withDefaultParsing {
+            
+            addNumberedListParsingWithMaxLevel(0, leadFormattingBlock: { (attributedString, range, level) in
+                var listString = ""
```

Comment:
You may also want to create a private static let for this since you use it in multiple places.